aviary/utils/functions.py

```diff
@@ -302,10 +302,16 @@ def setup(self):
             for variable_name in all_inputs:
                 units = get_units(variable_name)
                 if ':' in variable_name:
-                    add_aviary_input(self, variable_name, units=units)
+                    try:
+                        add_aviary_input(self, variable_name,
+                                         units=units, shape=num_nodes)
+                    except TypeError:
```

Comment:
I kind of wonder if this will cover up errors that we might want to catch while building components.

Reply:
I think the issue was that sometimes the default value in the meta_data was `None` which wasn't valid for the default value of add_input. This is just a print_comp, so the values aren't used in any calculations and are just printed to the terminal. I think the only thing you lose by falling back to the regular add_input is the default value from the meta data and the variable description from the meta data.